sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqllogictest/Main.java

```diff
@@ -44,6 +44,7 @@ public class Main {
     @SuppressWarnings("SpellCheckingInspection")
     public static void main(String[] argv) throws IOException {
         List<String> files = Linq.list(
+                "test/random/select/slt_good_58.test"
```

Comment:
why is only this one here?

Reply:
I have commented this out in another tiny PR.

Reply:
But the short answer is that the first failing test is here. There's an open issue for it.